lib/template-mixins.js

```diff
@@ -410,6 +427,8 @@ module.exports = function (fields, options) {
             var handler = _.isFunction(mixin.handler) ? mixin.handler : function () {
                 return function (key) {
                     key = extractKey.call(this, key);
+                    this.options = this.options || {};
+                    this.options.fields = this.options.fields || {};
```

Comment:
Is it worth passing in field as an option as well rather than setting in the renderWith functions?